### ISSUE TITLE
Light shadow when using anonymous auth option and dark theme.

### ISSUE THREAD

**Issue by austinhodak** [firebase/FirebaseUI-Android#1476]
```
<style name="SignInTheme" parent="Theme.AppCompat">
        <item name="colorPrimary">@color/timelineBG</item>
        <item name="colorPrimaryDark">@color/timelineDark</item>
        <item name="colorAccent">@color/md_orange_500</item>

        <item name="colorControlNormal">@color/md_orange_500</item>
        <item name="colorControlActivated">@color/md_orange_A700</item>
        <item name="colorControlHighlight">@color/md_orange_A200</item>
        <item name="android:windowBackground">@color/timelineBG</item>
    </style>
```

As you can see the rest of the button shadows are styled corrected other than the new anonymous one.

![smartselect_20181009-122203_battle buddy](https://user-images.githubusercontent.com/5241478/46683814-25e8b580-cbbf-11e8-9c0e-a09aef6c1c4e.jpg)

**Comment by samtstern** [comment]
Thanks @austinhodak for catching this!

cc @lsirac

**Comment by samtstern** [comment]
This has been fixed and released in version `4.2.1`.
